react-ui/src/pages/entry/EntryAppBarMain.jsx

```diff
@@ -1,26 +1,49 @@
 /**
  * Entry-specific content for top app bar
  */
+import { Typography } from '@mui/material';
```

Comment:
You shouldn't import directly from `@mui/material` but instead from the direct package, `import Typography from '@mui/material/Typography';`. I am not sure if we have set up the babel or if CRA has that built in, so this is a way to ensure that bundle sizes remain small. See https://mui.com/material-ui/guides/minimizing-bundle-size/ for more info.

There was an eslint rule that enforced this for the old version of MUI and should be updated for the new version. On a side note, but since we're already on the topic, do you remember why we've added the new version? We've increased the bundle size so users will have to download the code for both v4 and v5 to use the site. I'd ideally have liked to not introduce 5 until we've migrated away from 4 completely.

Reply:
I wish I could remember. I had thought it was to do with the Skeleton feature, but that seems to have come from V4.

I've added a personal note to migrate, we should aim to do that anyways because it's always a pain when you get 1-2 versions behind the docs.

Reply:
Yeah, I'd started a PR earlier following the migration steps, but never got around to finishing it. I probably won't tackle it again anytime soon. For this PR, I think if we can just change the `import { Component } from '@mui/material';` to `import Component from '@mui/material/Component';` we can deal with the migration later.